admin/console/hidden-settings.rst

```diff
@@ -96,13 +113,16 @@ Get the current state of this setting with:
 
    >> Setting.get('ui_user_organization_selector_with_email')
 
-
 Change Font-Settings for outgoing HTML-Mails
```

Comment:
"font settings" instead of "Font-Settings". "HTML mails" instead of "HTML-Mails".